packages/transaction-controller/src/utils/swaps.test.ts

```diff
@@ -375,6 +375,60 @@ describe('updateSwapsTransaction', () => {
       swapAndSendRecipient,
     });
   });
+
+  it('should update swap and send approval transaction and publish TransactionController:transactionNewSwapAndSendApproval', async () => {
```

Comment:
[nit] can remove async
```suggestion
  it('should update swap and send approval transaction and publish TransactionController:transactionNewSwapAndSendApproval', () => {
```